src/content/en/2021/ecommerce.md

```diff
@@ -551,9 +564,11 @@ The median ecommerce site on mobile made 30 requests to third parties. While las
 
 The weight of third-party content is also very similar to last year's analysis. With sites in the 50th percentile requesting 495 KB of third-party content. The bottom 10% requested 75 KB while the top 10% requested 2306 KB.
 
-## Tools {#tools}
+## Tools
 
-### JavaScript Frameworks & Libraries {#javascript-frameworks-&-libraries}
+{# TODO We don't use stacked headers. Can you add an intro sentence as to what this section is about? #}
```

Comment:
```suggestion
In addition to site performance & quality analysis, our [Methodology](./methodology) enables us to review other technologies used on ecommerce sites. This provides us with insight into the ecommerce strategies adopted, e.g. internationalisation, as well as typical development techniques, e.g. JavaScript libraries used. 
```

Reply:
Looks good, but tweaked it very slightly if that's OK.

```suggestion
In addition to site performance and quality analysis, our [Methodology](./methodology) enables us to review other technologies used on ecommerce sites. This provides us with insight into the ecommerce strategies adopted (e.g., internationalization), as well as typical development techniques (e.g., JavaScript libraries used).
```